NFIQ2/NFIQ2Api/nfiq2api.cpp

```diff
@@ -39,19 +39,25 @@ extern "C" {
     *ocv = buf;
 #endif
   }
-  DLLEXPORT void STDCALL InitNfiq2()
+  DLLEXPORT const char* STDCALL InitNfiq2()
   {
     try
     {
       if( g_nfiq2.get() == nullptr )
       {
+#       ifdef EMBED_RANDOMFOREST_PARAMETERS
         g_nfiq2 = std::unique_ptr<NFIQ::NFIQ2Algorithm>( new NFIQ::NFIQ2Algorithm() );
+#       else
+        g_nfiq2 = std::unique_ptr<NFIQ::NFIQ2Algorithm>( new NFIQ::NFIQ2Algorithm("nfiq2rf.yaml", "0xccd75820b48c19f1645ef5e9c481c592") );
```

Comment:
Seems unusual to embed the `0x` prefix here. Suggest we take this out in a revision.